tests/notifier/notifiers/inventory_summary_test.py

```diff
@@ -312,7 +362,8 @@ def test_inventory_summary_can_run_successfully(self):
         mock_session.query.return_value.get.return_value = mock_inventory_index
 
         mock_service_config = mock.MagicMock()
-        mock_service_config.scoped_session.return_value.__enter__.return_value = mock_session
+        mock_service_config.scoped_session.return_value.__enter__.return_value \
```

Comment:
Try this instead:
```
        mock_service_config.scoped_session.return_value.__enter__.return_value = (
            mock_session)
```

Reply:
Above line is too long - have just left as is.